vcd/config_test.go

```diff
@@ -110,6 +110,8 @@ type TestConfig struct {
 		MediaPath                    string `json:"mediaPath"`                    // Media path, if different from Media.MediaPath
 		MediaName                    string `json:"mediaName"`                    // Media name to create
 		OvaPath                      string `json:"ovaPath"`                      // Ova Path, if different from Ova.OvaPath
+		OrgUser                      string `json:"orgUser"`                      // Org User to be created within the organization
```

Comment:
I considered that, but then I thought that the purpose of this field is to allow us to create the environment the way we want it, because after that this is the user that we connect with when running tests as non-sysAdmin.
Moreover, you know that we have a tool that creates the environment for you. If you leave it to de defaults, the user will be `org_name-admin`.